src/multi/mod.rs

```diff
@@ -37,6 +39,7 @@ use core::num::NonZeroUsize;
 /// ```
 #[cfg(feature = "alloc")]
 #[cfg_attr(feature = "docsrs", doc(cfg(feature = "alloc")))]
+#[deprecated = "Replaced with `many`"]
```

Comment:
I'd like to hold out on the deprecation warnings. `many0` and others are extensively used, I'd rather keep them around in the next major version and ease transition to range based combinators

Reply:
I figured deprecating would direct people towards the new parsers while remaining fully backwards compatible. But I can take out the warnings.

Reply:
Done